plugins/modules/tenant_group.py

```diff
@@ -43,25 +43,27 @@
       - Name of the tenant group to be created
     required: true
     type: str
-    version_added: "3.0.0"
```

Comment:
I believe we need to keep these. And prob want to add this to the `parent_tenant_group` option as 3.1.0

Reply:
Yes, these are a must, and `parent_tenant_group` **must** have the option added. This is a requirement from Ansible to be included in the base release of Ansible.

Reply:
OK, check that now.